core/trading_engine.py

```
@@ -673,61 +960,164 @@ async def _execute_signals(self, entry_signals: List[Signal], exit_signals: List
+
+                # ===== 레버리지 설정 (주문 전 필수) =====
+                leverage = exchange_config.leverage
+                logger.info(f"[{exchange_name}] 레버리지 설정 시도: {signal.symbol} = {leverage}x")
+                try:
+                    if hasattr(connector, 'set_leverage'):
+                        leverage_result = connector.set_leverage(signal.symbol, leverage)
+                        if leverage_result:
+                            logger.info(f"✅ 레버리지 설정 성공: {signal.symbol} = {leverage}x")
+                        else:
+                            logger.warning(f"⚠️ 레버리지 설정 실패 (이미 설정됨 가능): {signal.symbol}")
+                except Exception as lev_error:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)